src/com/github/mtdrewski/GRAPH_moment/controller/Menu.java

```diff
@@ -0,0 +1,51 @@
+package com.github.mtdrewski.GRAPH_moment.controller;
+
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.image.Image;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Text;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+import java.util.function.Supplier;
+
+public class Menu {
+    @FXML
+    private VBox menu;
+
+    private Supplier<Stage> dialogFactory = () -> {
+        Stage dialog = new Stage();
+        dialog.initModality(Modality.APPLICATION_MODAL);
+        dialog.initOwner((Stage) menu.getScene().getWindow());
+        dialog.getIcons().add(new Image("icon.png"));
+        return dialog;
+    };
+
+    public void newProjectOnClick(ActionEvent event) throws Exception {
```

Comment:
Does it throw just any Exception? (Maybe use more specific Exception mark)

Reply:
It should probably be IOException, but so far I'm not sure if it should be thrown and not catch. Moreover newProjectOnClick isn't finished, so just ignore it for now.

Reply:
ok, so just mark it as //TODO or something